openapi/preprocess_spec.py

```diff
@@ -330,20 +330,27 @@ def main():
     )
     args = argparser.parse_args()
 
-    spec_url = 'https://raw.githubusercontent.com/%s/%s/' \
+
+    skip_fetch_spec = "OPENAPI_SKIP_FETCH_SPEC"
```

Comment:
I'd remove this and just inline it into the `os.environ` call below...

Reply:
done